orion-server/src/main/java/com/pinterest/orion/server/OrionServer.java

```diff
@@ -302,11 +302,15 @@ public static void main(String[] args) throws Exception {
   }
 
   private static MetricName constructMetricsName(String subject, String action, String outcome, Map<String, String> tagMap) {
```

Comment:
And have a single param called `metricName` here instead of `String subject, String action, String outcome`
Action and outcome are not for Orion to decide.